build-bench-env.sh

```diff
@@ -343,6 +353,16 @@ function checkout {  # name, git-tag, git repo, options
   write_version $1 $2 $3
 }
 
+function check_checksum {  # name, sha256sum
+  if (echo "$2  $1" | $SHA256SUM_CMD --check --status); then
+    echo "$1 has correct checksum"
+  else
+    echo "$1 has wrong checksum"
+    echo "$2  was expected"
```

Comment:
```suggestion
    echo "$2 was expected"
```

Reply:
Was purely aesthetic, so that "was expected" is aligned with the filename on next line, as there are two spaces between checksum and file. Will fix it!